src/PersonObjects/Sleeve/ui/TaskSelector.tsx

```diff
@@ -226,8 +227,18 @@ function getABC(sleeve: Sleeve): [string, string, string] {
       return ["Commit Crime", sleeve.crimeType, "------"];
     case SleeveTaskType.Class:
       return ["Take University Course", sleeve.className, sleeve.currentTaskLocation];
-    case SleeveTaskType.Gym:
-      return ["Workout at Gym", sleeve.gymStatType, sleeve.currentTaskLocation];
+    case SleeveTaskType.Gym: {
+      const sanitizedStat: string = sleeve.gymStatType.toLowerCase();
+      if (sanitizedStat.includes("str")) {
+        return ["Workout at Gym", "Train Strength", sleeve.currentTaskLocation];
+      } else if (sanitizedStat.includes("def")) {
+        return ["Workout at Gym", "Train Defense", sleeve.currentTaskLocation];
+      } else if (sanitizedStat.includes("dex")) {
+        return ["Workout at Gym", "Train Dexterity", sleeve.currentTaskLocation];
+      } else if (sanitizedStat.includes("agi")) {
+        return ["Workout at Gym", "Train Agility", sleeve.currentTaskLocation];
+      }
+    }
```

Comment:
While this does kind of work, it doesn't solve the real problem, which is that `Sleeve.workoutAtGym()` stores arbitrary user input in `Sleeve.gymStatType` field that then needs to be identified over and over again wherever it is used.

The correct solution is to normalise the user input to one of an enumerated list of strings (eg. `"", "str", "def", "dex", "agi"`), and then other code that uses `Sleeve.gymStatType` can rely on it being one of those known values.

Reply:
Agreed. I'll get back to it if I have some time to spare.

If not... well it is working, and resolve an actual issue. It will be shipped as is.

Reply:
Done !